src/panel/poi_panel.js

```diff
@@ -89,6 +89,7 @@ PoiPanel.prototype.closeAction = function() {
   if (this.poi.meta && this.poi.meta.source) Telemetry.add("close", "poi", this.poi.meta.source)
   fire('clean_marker')
   PanelManager.resetLayout()
+  fire("restore_location")
```

Comment:
I am not sure we should try to restore a location on close. Especially when the poi panel is not related to any list.

Reply:
Sounds fare. I removed it.